app/components/termsoverlay/termsoverlay.js

```diff
@@ -13,82 +13,217 @@
+        <form className='terms-overlay-form'>
+          {agreeConfirmation}
+          {backBtn}
+          <button
+            className='terms-button terms-button-submit'
+            onClick={this.handleTermsAndPrivacySubmit}
+            disabled={continueBtnDisabled}>Continue</button>
+        </form>
+      </div>
+    );
+  },
+  renderBackBtn:function(){
     return (
       <button
-        className="btn btn-primary js-terms-submit"
-        onClick={this.handleSubmit}
-        disabled={disabled}>Continue</button>
+        className='terms-button terms-button-back'
+        onClick={this.handleBack}>Back</button>
     );
-    
   },
+  renderAgreeCheckboxes:function(){
+
+    var onBehalf;
+    if (this.state.ageSelected === this.props.ages.WITH_CONSENT.value) {
+      onBehalf = (
+        <label htmlFor='agreedOnBehalf'>
+          <input
+            id='agreedOnBehalf'
+            type='checkbox'
+            className='js-terms-checkbox'
+            checked={this.state.agreedOnBehalf}
+            onChange={this.handleOnBehalfAgreementChange} />
+          {this.props.messages.ACCEPT_ON_BEHALF}
+        </label>
+      );
+    }
 
-  renderCheckbox: function() {
-    
     return (
-      <div className="terms-overlay-form-checkbox">
-        <label htmlFor="agreed">
+      <div className='terms-overlay-accept-checkbox'>
+        <label htmlFor='agreed'>
           <input
-            id="agreed"
-            type="checkbox"
-            className="js-terms-checkbox"
+            id='agreed'
+            type='checkbox'
+            className='js-terms-checkbox'
             checked={this.state.agreed}
-            onChange={this.handleChange} />
-          <span> I agree to these terms</span>
+            onChange={this.handleAgreementChange} />
+          {this.props.messages.ACCEPT_OF_AGE}
         </label>
+        {onBehalf}
       </div>
     );
-    
   },
-
-  handleChange: function(e) {
-    var checked = e.target.checked;
-    this.setState({agreed: checked});
+  renderSorryMessage:function(){
+    var backBtn = this.renderBackBtn();
+    return (
+      <div ref='sorryMsg'>
+        <p className='terms-overlay-sorry-message'>{this.props.messages.SORRY_NOT_OF_AGE}</p>
+        {backBtn}
+      </div>
+    );
   },
+  render: function() {
+
+    var content = this.renderAgeConsentStep();
+
+    if(this.state.ageConfirmed && this.state.ageSelected !== this.props.ages.NOT_OF_AGE.value){
+      content = this.renderTermsAndPrivacyStep();
+    }else if( this.state.ageConfirmed && this.state.ageSelected === this.props.ages.NOT_OF_AGE.value){
+      content = this.renderSorryMessage();
+    }
+
+    return (
+      <div className='terms-overlay js-terms'>
+        <LoginNav hideLinks={true} trackMetric={this.props.trackMetric} />
+        <div className='terms-overlay-content terms-overlay-box'>
+          {content}
+        </div>
+      </div>
+    );
 
-  handleSubmit: function(e) {
+  },
+  websiteTerms: function() {
```

Comment:
yeah had same thought but will leave up to @HowardLook and @brandonarbiter to decide if its good enough